packages/react-components/src/MessageSignature.tsx

```diff
@@ -2,11 +2,11 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { ContractABIMessage } from '@polkadot/api-contract/types';
+import { AbiMessage } from '@polkadot/api-contract/types';
```

Comment:
Can we get this type through cennznet/api or change cennznet api to provide this too?

Reply:
I think this is completely different sub package(api-contract) of the mono repo... I would avoid adding these types in API as of yet till we start supporting...